### PR TITLE
Add Record Screen With Keystroke Pro command

### PR DESCRIPTION
**Pull request by danulqua**
## Description

Some people love to use the Keystroke Pro app to display keystrokes during screen recordings but don't like keeping it running in the background all the time. This script command simultaneously launches Keystroke Pro and starts CleanShot's Screen Recording.

## Type of change

- [x] New script command

## Screenshot

Raycast 2023-08-12 at 12 46 03

## Dependencies / Requirements

CleanShot X and Keystroke Pro should be installed.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/cleanshot/cleanshot-record-screen-with-keystroke-pro.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Optional parameters:
+# @raycast.icon ./images/record-screen-keystroke-pro.png
+# @raycast.packageName CleanShot X
+
+# Documentation:
+# @raycast.author danulqua
+# @raycast.authorURL https://raycast.com/danulqua
+# @raycast.description Launch Keystroke Pro and CleanShot X screen recording.
+# @raycast.description Start a screen recording with keystrokes using the Keystroke Pro app and save it as a video or an optimized GIF file.
```
Comment (unnamedd) [inline commands/apps/cleanshot/cleanshot-record-screen-with-keystroke-pro.sh:22]: You cannot have 2 `@raycast.description`s, you must keep everything you want to describe at the same metatag.

```suggestion
# @raycast.description Launch Keystroke Pro and CleanShot X screen recording. Start a screen recording with keystrokes using the Keystroke Pro app and save it as a video or an optimized GIF file.
```

File: commands/apps/cleanshot/cleanshot-record-screen-with-keystroke-pro.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# @raycast.schemaVersion 1
+# @raycast.title Record Screen With Keystroke Pro
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/record-screen-keystroke-pro.png
+# @raycast.packageName CleanShot X
+
+# Documentation:
+# @raycast.author danulqua
```
Comment (unnamedd) [inline commands/apps/cleanshot/cleanshot-record-screen-with-keystroke-pro.sh:19]: Will be nicer if you use your own name here, to make it visible in the tools and also on the Awesome Script Commands we do have (this is the README.md)

```suggestion
# @raycast.author Danylo Zalizchuk
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @danulqua, thanks for your contribution! 
I've made some comments here, nothing problematic, but a nice to have. 

Please, let us know when you finish to address them, will be a pleasure to have your contribution in our repository.

File: commands/apps/cleanshot/cleanshot-record-screen-with-keystroke-pro.sh (hunk 1 above)
Comment (danulqua) [reply commands/apps/cleanshot/cleanshot-record-screen-with-keystroke-pro.sh]: @unnamedd oh, of course! I just forgot to remove one of them. Fixed!

**Comment by danulqua** [comment]
Hey @unnamedd, letting you know that I made changes according to your suggestions. Waiting for a review :)

**Review by unnamedd — APPROVED** [review]
Hey @danulqua, thanks for the changes and sorry for the delay in the answer! 
It looks good to me! It is approved! 

Our friends from Raycast will merge as soon as possible! ;)

**Pull request merged**